packages/11ty/_plugins/iiif/process/manifest/index.js

```diff
@@ -0,0 +1,56 @@
+const { globalVault } = require('@iiif/vault')
+const vault = globalVault()
+const { IIIFBuilder } = require('iiif-builder')
+const builder = new IIIFBuilder(vault)
+const Annotation = require('./annotation')
+const Base = require('./base')
+const Canvas = require('./canvas')
+const Choices = require('./choices')
```

Comment:
Separating `require` statements that act as an `import` from other `const` declarations.
```suggestion
const { globalVault } = require('@iiif/vault')
const { IIIFBuilder } = require('iiif-builder')
const Annotation = require('./annotation')
const Base = require('./base')
const Canvas = require('./canvas')
const Choices = require('./choices')

const vault = globalVault()
const builder = new IIIFBuilder(vault)
```